pkg/specs/chart.go

```diff
@@ -19,8 +19,9 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-func DefaultHelmRelease(chartPath string) api.Spec {
-	return api.Spec{
+func (r *Resolver) DefaultHelmRelease(chartPath string) api.Spec {
+	noOutro := r.Viper.GetBool("no-outro")
```

Comment:
I think in general we prefer to set these as fields on `(r *Resolver)` where possible in the `New` function, rather than accessing `r.Viper` inside a method. 

For example, check out https://github.com/replicatedhq/ship/blob/master/pkg/ship/ship.go#L79

Reply:
Yeah, that's my fault... still probably the better pattern to follow though @10Dimensional